analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CreatingHashAlgorithms.cs

```diff
@@ -1,77 +1,104 @@
-﻿namespace NS
-{
-    using System.Security.Cryptography;
+﻿using System.Security.Cryptography;
 
-    public class TestClass
+namespace Tests.Diagnostics
+{
+    public class InsecureHashAlgorithm
     {
-        // RSPEC 4790: https://jira.sonarsource.com/browse/RSPEC-4790
-        public void ComputeHash()
+        public void Hash(byte[] temp)
         {
-            // Review all instantiations of classes that inherit from HashAlgorithm, for example:
-            HashAlgorithm hashAlgo = HashAlgorithm.Create();
```

Comment:
This overload became safe now? That's suspicious. Should we document that it's compliant now in a test case?

Reply:
S2070 does not check for this and I've checked already with @eric-therond-sonarsource and we should mirror it's implementation but sure I can add a test case to document the behavior.